### DISCUSSION TITLE
Arkaine's Valor missing

### DISCUSSION THREAD

**Discussion by Drav1417** [diasurgical/DevilutionX#6302] [Q&A]
Howdy guys, just started playing yesterday and love the work you've done, it's been a mostly flawless experience thus far but I may have run into a problem with Arkaine's Valor not dropping. Read the book, got the blood stones, opened up the final passageway and I just got a random chest piece with +Light Radius and the quest is still open. I couldn't see anything relating to this particular issue so if I missed something obvious then apologies in advance.

**Comment by StephenCWills** [comment]
As of 1.5.0, SP quests now only drop their quest-specific rewards in Normal difficulty.

**Reply by AJenbo** [reply]
The quest should have completed when you picked it up though.

**Reply by Drav1417** [reply]
Ah damn. I'm not playing Nightmare as a NG+, I started with it since it was available. Any chance that might become a gameplay option instead in a future update?

@AJenbo yeah it's still active in my quest log unfortunately. EDIT - Appears to be the same with the Halls of the Blind quest. Got the amulet but it's still an active quest. Is it to do with the quest item being replaced with a random one?
